internal/driver/glfw/menu_bar_test.go

```diff
@@ -54,7 +54,7 @@ func TestMenuBar(t *testing.T) {
 
 		menuBar := glfw.NewMenuBar(menu, c)
 		themeCounter := 0
-		button := widget.NewButton("Button", func() {
+		button := newNotFocusableButton("Button", func() {
```

Comment:
I think now this is unneeded and we can use widget.NewButton again IMHO.

Reply:
Can you expand on this reasoning please? I added it because it was needed, so a little more help as to why you think it is not needed would be appreciated.

Reply:
> Can you expand on this reasoning please? I added it because it was needed, so a little more help as to why you think it is not needed would be appreciated.

I replaced it locally, and it just look like before, nothing seems to be broken 🤔. I think you tried that solution because previously widgets that are tappable and focusable, were focused when tapped, but now it isn't the case, right?

Reply:
No, it breaks the keyboard focus cycle - the menu and button are on the same focus manager in this test and tapping "left, left" should wrap to the right menu, then the middle one, but instead it now hits the button first then the right hand menu. So I made it not-a-focusable-button instead.

Reply:
Oh I see, you are right, sorry